notebooks/curation_api/python/src/utils/config.py

```diff
@@ -1,39 +1,64 @@
+import json
 import os
 
+import boto3
 import requests
 
 
-def set_api_urls(env: str) -> None:
+def set_api_urls(env: str, stack: str = "") -> None:
```

Comment:
Yeah I had a brief discussion with myself about that and decided it was fine as-is because it's unexpectedness defaults to a "safe" behavior. If it was the other way around, and adding `env="prod"` overrode the `stack` argument then I would agree it is a problem 😅 .

Not going to bother to rework due to internal tool status, as you reference.